ikenna-portfolio/portfolio/target/portfolio-1/style.css

```diff
@@ -4,20 +4,78 @@
   width: 75%;
 }
 
-.image-format {
-  border: 10px solid black;
-  max-width: 80%;
+#quote-container {
+  margin-top: 20px;
 }
 
-.row {
-  display: flex;
+.bg-1 { 
+  background-color: #7a5b31;
+  color: #ffffff;
+}
+  
+.bg-2 { 
+  background-color: #8d8741;
+  color: #ffffff;
+}
+  
+.bg-3 { 
+  background-color: #b89b1a;
+  color: #070101;
 }
 
+.bg-4 { 
+  background-color: #fbeec1;
+  color: #555555;
+}
+
+.bg-5 { 
+  background-color: #bc986a;
+  color: #ffffff;
+}
+  
 .column {
   flex: 1 1 auto;
   padding: 5px;
 }
 
-#quote-container {
-  margin-top: 20px;
+.container-fluid {
+  padding-bottom: 70px;
```

Comment:
This could be changed to padding: 70px 0;

and similar change below.

Reply:
done